test/Microsoft.IdentityModel.TestUtils/KeyingMaterial.cs

```diff
@@ -1057,5 +1071,19 @@ public static JsonWebKey JsonWebKeyX509_2048_As_RSA_With_KeyId
                 return jsonWebKey;
             }
         }
+
+        private static SecureString ConvertToSecureString(string password)
+        {
+            if (password == null)
+                throw new ArgumentNullException("password");
```

Comment:
nit: can be replace with nameof(password)

Reply:
Fixed.